openbb_platform/providers/intrinio/openbb_intrinio/models/calendar_ipo.py

```diff
@@ -0,0 +1,190 @@
+    )
+    share_count: Optional[int] = Field(
+        description="The number of shares offered in the IPO.", default=None
+    )
+    share_count_lowest: Optional[int] = Field(
+        description="""
+            The expected lowest number of shares that will be offered in the IPO. Before an IPO is priced,
+            companies typically provide a range of shares that they expect to offer in the IPO
+            (typically available for upcoming IPOs).
+        """,
+        default=None,
+    )
+    share_count_highest: Optional[int] = Field(
+        description="""
+            The expected highest number of shares that will be offered in the IPO. Before an IPO is priced,
+            companies typically provide a range of shares that they expect to offer in the IPO
+            (typically available for upcoming IPOs).
+        """,
+        default=None,
+    )
+    announcement_url: Optional[str] = Field(
+        description="The URL to the company's announcement of the IPO", default=None
+    )
+    sec_report_url: Optional[str] = Field(
+        description="""
+            The IRL to the company's S-1, S-1/A, F-1, or F-1/A SEC filing,
+            which is required to be filed before an IPO takes place.
+        """,
+        default=None,
+    )
+    open_price: Optional[float] = Field(
+        description="The opening price at the beginning of the first trading day (only available for priced IPOs).",
+        default=None,
+    )
+    close_price: Optional[float] = Field(
+        description="The closing price at the end of the first trading day (only available for priced IPOs).",
+        default=None,
+    )
+    volume: Optional[int] = Field(
+        description="The volume at the end of the first trading day (only available for priced IPOs).",
+        default=None,
+    )
+    day_change: Optional[float] = Field(
+        description="""
+            The percentage change between the open price and the close price on the first trading day
+            (only available for priced IPOs).
+        """,
+        default=None,
+    )
+    week_change: Optional[float] = Field(
+        description="""
+            The percentage change between the open price on the first trading day and the close price approximately
+            a week after the first trading day (only available for priced IPOs).
+        """,
+        default=None,
+    )
+    month_change: Optional[float] = Field(
+        description="""
+            The percentage change between the open price on the first trading day and the close price approximately
+            a month after the first trading day (only available for priced IPOs).
+        """,
+        default=None,
+    )
+    id: Optional[str] = Field(description="The Intrinio ID of the IPO.", default=None)
+    company: Optional[IntrinioCompany] = Field(
+        description="The company that is going public via the IPO.", default=None
+    )
+    security: Optional[IntrinioSecurity] = Field(
```

Comment:
These return fields are dictionaries, is is possible to unnest them?
<img width="882" alt="Screenshot 2023-10-25 at 1 43 24 PM" src="https://github.com/OpenBB-finance/OpenBBTerminal/assets/18151143/db47a4e1-4062-4c2c-bcbb-b3dd63b2da9c">

Reply:
> These return fields are dictionaries, is is possible to unnest them? <img alt="Screenshot 2023-10-25 at 1 43 24 PM" width="882" src="https://user-images.githubusercontent.com/18151143/278100770-db47a4e1-4062-4c2c-bcbb-b3dd63b2da9c.png">

If you look at the response under `results`, they are actually the validated models.